feat(apollo-client): the spi of config service load balancer client (#4394)
apolloconfig/apollo: docs/en/usage/java-sdk-user-guide.md

}
```

# Ⅶ. apollo-client customization

## 7.1 ConfigService load balancing algorithm

> from version 2.1.0

> For satisfy user's different demands on ConfigService load balancing algorithm when user use apollo-client, we provide **spi** in version 2.1.0

Reviewer: ```suggestion
To satisfy users' different demands on ConfigService load balancing algorithm when using apollo-client, we provide **spi** since version 2.1.0
```

The interface is `com.ctrip.framework.apollo.spi.ConfigServiceLoadBalancerClient`.

> The Input is multiple ConfigServices returned by meta server, and the output is a ConfigService.

Reviewer: ```suggestion
The Input is multiple ConfigServices returned by meta server, and the output is a ConfigService selected.
```

> The default service provider is `com.ctrip.framework.apollo.spi.RandomConfigServiceLoadBalancerClient`，use random strategy which choose one ConfigService from multiple ConfigServices.

Reviewer: ```suggestion
The default service provider is `com.ctrip.framework.apollo.spi.RandomConfigServiceLoadBalancerClient`, which chooses one ConfigService from multiple ConfigServices using random strategy .
```
